### PR TITLE
Making instance creation a bit more extensible

### PR DESCRIPTION
**Pull request by hub-cap**

### CODE REVIEW

**Comment by TimSimpsonR** [comment]
+1

File: reddwarf/instance/service.py [hunk 1]
```diff
@@ -163,8 +163,11 @@ def create(self, req, body, tenant_id):
         LOG.info(_("req : '%s'\n\n") % req)
         LOG.info(_("body : '%s'\n\n") % body)
         context = req.environ[wsgi.CONTEXT_KEY]
-        database = models.ServiceImage.find_by(service_name='mysql')
-        image_id = database['image_id']
+        service_type = body['instance'].get('service_type')
```
Comment (rnirmal) [inline reddwarf/instance/service.py:166]: `service_type = body['instance'].get('service_type', "mysql")`

Comment (rnirmal) [inline reddwarf/instance/service.py:166]: Ignore it... we might need to validate that in the future. so it won't even get to this step with none.

**Comment by rnirmal** [comment]
+1

**Pull request merged**